uiowa/uiowa
# Upgrade dev dependencies to stable release if possible and lock others.

Resolves #2793 
Resolves #2593
Resolves #2910
Finally resolves #2474 

### How to test
- Review https://github.com/uiowa/uiowa/pull/2912/files
  - Review the updates to the readme about updating drupal/core and contrib packages.
    - This includes removal of `drupal/core` which resulted in some updates: https://github.com/uiowa/uiowa/pull/2912/commits/9c38c9ecf20e4a696409aeb776b76e4294a1947f
  - Review the readme section on locked dependencies.
  - Review the changes in the composer.json file.
- Pull and run `composer install`.
  - Check the Composer output for anything unusual...
  - Note other updates that might need additional testing and outline those here.
- Test out the menu_block update.
  - Sync sites with block config which includes menu_block settings: default, hr.uiowa.edu
    -  **Need help** here with testing criteria.
  - Some directories have block config without menu_block settings which were not affected: sitenow.uiowa.edu, policy.clas.uiowa.edu, uids_base/config/optional, 
- TBD.
